### PR TITLE
[scriptable-ios]: Update for Scriptable app v1.5.0

### PR DESCRIPTION
**Pull request by schl3ck**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://docs.scriptable.app (no changelog there) or https://apps.apple.com/us/app/scriptable/id1405459188 (see _Version history_)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

-----
`npm run lint scriptable-ios` gives me an error `The 'no-redundant-jsdoc-2' rule threw an error in <file>: Unexpected tag kind: JSDocSeeTag`. Does this mean that the `@see` tag is not allowed? If yes, what is the reason?

### CODE REVIEW

File: types/scriptable-ios/index.d.ts [hunk 1]
```diff
@@ -5274,6 +5770,166 @@ declare class WebView {
...
+    /**
+     * _Left aligns the image._
+     *
+     * Specifies that image should be left aligned. This is the default.
+     * @see https://docs.scriptable.app/widgetimage/#-leftalignimage
+     */
+    leftAlignImage(): void;
+
+    /**
+     * _Left aligns the image._
```
Comment (FuJuntao) [inline types/scriptable-ios/index.d.ts:5841]: This should be "Center", right?

File: types/scriptable-ios/index.d.ts [hunk 2]
```diff
@@ -5274,6 +5770,166 @@ declare class WebView {
...
+     *
+     * Specifies that image should be left aligned.
```
Comment (FuJuntao) [inline types/scriptable-ios/index.d.ts:5843]: Here is "center" too.

File: types/scriptable-ios/index.d.ts [hunk 3]
```diff
@@ -5274,6 +5770,166 @@ declare class WebView {
...
+    /**
+     * _Left aligns the text._
+     *
+     * Specifies that text should be left aligned. This is the default.
+     * @see https://docs.scriptable.app/widgettext/#-leftaligntext
+     */
+    leftAlignText(): void;
+
+    /**
+     * _Left aligns the text._
```
Comment (FuJuntao) [inline types/scriptable-ios/index.d.ts:5917]:
```suggestion
     * _Center aligns the text._
```

File: types/scriptable-ios/index.d.ts [hunk 4]
```diff
@@ -5274,6 +5770,166 @@ declare class WebView {
...
+     *
+     * Specifies that text should be left aligned.
```
Comment (FuJuntao) [inline types/scriptable-ios/index.d.ts:5919]:
```suggestion
     * Specifies that text should be center aligned.
```

**Review by FuJuntao — CHANGES_REQUESTED** [review]

**Comment by schl3ck** [comment]
Thank you @FuJuntao for spotting the typos!

I've checked the whole file for types now.

**Review by FuJuntao — APPROVED** [review]
Looking good to me! By the way, may I ask when will this be released?

**Comment by schl3ck** [comment]
Can somebody please merge this anytime soon?

**Comment by schl3ck** [comment]
@FuJuntao Can you please review the new changes?

**Review by FuJuntao — APPROVED** [review]

**Comment by schl3ck** [comment]
Ready to merge

**Pull request merged**